File: dsa/02-blind-75/arrays-hashing/01-two-sum/solutions/python/two_sum_brute_force.py
def two_sum(numbers: list[int], target: int) -> list[int]:
    for left in range(len(numbers)):
        for right in range(left + 1, len(numbers)):
            if numbers[left] + numbers[right] == target:
                return [left, right]
    raise ValueError("No pair adds to the target")

File: dsa/02-blind-75/arrays-hashing/01-two-sum/solutions/typescript/twoSumBruteForce.ts
export function twoSum(numbers: number[], target: number): [number, number] {
  for (let left = 0; left < numbers.length; left += 1) {
    for (let right = left + 1; right < numbers.length; right += 1) {
      if (numbers[left] + numbers[right] === target) {
        return [left, right];
      }
    }
  }

  throw new Error("No pair adds to the target");
}
